### ISSUE TITLE
cannot build debian 10 template

### ISSUE THREAD

**Issue by zaoqi** [QubesOS/qubes-issues#5419]
**Qubes OS version**
R4.0

**Affected component(s) or functionality**
_The component or functionality of Qubes OS that is not working as expected._

**Brief summary**
```
[user@work-template-dev qubes-builder]$ make template
'qubes-mgmt-salt_4.0.19-1+deb10u1_amd64.buildinfo' looks like architecture 'amd64', but this is not listed in the Architecture-Header!
Ignoring as --ignore=surprisingarch given.
-> Building template buster (logfile: build-logs/template-buster.log)...
make: *** [Makefile:319: template-local-buster+standard] Error 1
[user@work-template-dev qubes-builder]$ cat build-logs/template-buster.log 
Building template: debian-10
## lots of lines......
Setting up openssl (1.1.1d-0+deb10u2) ...
Setting up openssh-client (1:7.9p1-10+deb10u1) ...
Setting up e2fsprogs (1.44.5-1+deb10u2) ...
Processing triggers for libc-bin (2.28-10) ...
Processing triggers for man-db (2.8.5-2) ...
Removing 'local diversion of /sbin/initctl to /sbin/initctl.distrib'
-> Unmounting prepared_image...
--> Copying prepared_images/debian-10.img to qubeized_images/debian-10/root.img...
--> Mounting qubeized_images/debian-10/root.img
Adding 'local diversion of /sbin/initctl to /sbin/initctl.distrib'
Get:1 file:/tmp/qubes_repo buster InRelease
Ign:1 file:/tmp/qubes_repo buster InRelease
Get:2 file:/tmp/qubes_repo buster Release [340 B]
Get:2 file:/tmp/qubes_repo buster Release [340 B]
Get:3 file:/tmp/qubes_repo buster Release.gpg
Ign:3 file:/tmp/qubes_repo buster Release.gpg
Get:4 file:/tmp/qubes_repo buster/main amd64 Packages [14.6 kB]
Hit:5 https://cdn-aws.deb.debian.org/debian buster InRelease
Hit:6 https://cdn-aws.deb.debian.org/debian-security buster/updates InRelease
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 qubes-vm-dependencies : Depends: qubes-gui-agent but it is not going to be installed
                         Recommends: qubes-vm-recommended but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
Removing 'local diversion of /sbin/initctl to /sbin/initctl.distrib'
make[1]: *** [Makefile:64: rootimg-build] Error 100
```

**To Reproduce**

```
    1  git clone https://github.com/QubesOS/qubes-builder.git qubes-builder
    2  cd qubes-builder/
    3  ./setup 
    4  make install-deps
    5  make get-sources
    6  make qubes-vm
    7  make template
```

**Expected behavior**
_A clear and concise description of what you expected to happen._

**Actual behavior**
_What actually happened (instead of what you expected to happen)._

**Screenshots**
_If applicable, add screenshots to help explain your problem._

**Additional context**
_Add any other context about the problem here._

**Solutions you've tried**
_If applicable, any solutions or workarounds you've already tried._

**Relevant [documentation](https://www.qubes-os.org/doc/) you've consulted**

https://www.qubes-os.org/doc/building-archlinux-template/
https://www.qubes-os.org/doc/templates/ubuntu/

**Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**
_A list of links to other bug reports, feature requests, or tasks in the qubes-issues tracker (or "none" if you didn't find any). Do not describe any other unreported bugs, features, or tasks here._

**Comment by andrewdavidwong** [comment]
Duplicate of #4778

**Comment by andrewdavidwong** [comment]
This appears to be a [duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) of an existing issue. If so, please comment on the appropriate existing issue instead. If you believe this is not really a duplicate, please leave a comment briefly explaining why. We'll be happy to take another look and, if appropriate, reopen this issue. Thank you.

**Comment by unman** [comment]
On Sat, Oct 26, 2019 at 07:47:43AM -0700, Zaoqi wrote:
> **Qubes OS version**
> R4.0
> 
> **Affected component(s) or functionality**
> _The component or functionality of Qubes OS that is not working as expected._
> 
> **Brief summary**
> ```
> [user@work-template-dev qubes-builder]$ make template
> 'qubes-mgmt-salt_4.0.19-1+deb10u1_amd64.buildinfo' looks like architecture 'amd64', but this is not listed in the Architecture-Header!
> Ignoring as --ignore=surprisingarch given.
> -> Building template buster (logfile: build-logs/template-buster.log)...
> make: *** [Makefile:319: template-local-buster+standard] Error 1
> ```
> 
> **To Reproduce**
> 
> ```
>     1  git clone https://github.com/QubesOS/qubes-builder.git qubes-builder
>     2  cd qubes-builder/
>     3  ./setup 
>     4  make install-deps
>     5  make get-sources
>     6  make qubes-vm
>     7  make template
> ```
> 
> **Expected behavior**
> _A clear and concise description of what you expected to happen._
> 
> **Actual behavior**
> _What actually happened (instead of what you expected to happen)._
> 
> **Screenshots**
> _If applicable, add screenshots to help explain your problem._
> 
> **Additional context**
> _Add any other context about the problem here._
> 
> **Solutions you've tried**
> _If applicable, any solutions or workarounds you've already tried._
> 
> **Relevant [documentation](https://www.qubes-os.org/doc/) you've consulted**
> 
> https://www.qubes-os.org/doc/templates/archlinux/
> https://www.qubes-os.org/doc/templates/ubuntu/
> 
> **Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**
> _A list of links to other bug reports, feature requests, or tasks in the qubes-issues tracker (or "none" if you didn't find any). Do not describe any other unreported bugs, features, or tasks here._
> 
> 
Mail to qubes-users list and include relevant part of the log file.

**Comment by unman** [comment]
@andrewdavidwong Actually this isn't a duplicate because #4778 was about stretch and this is debian-10 (buster) , but I dont think either of them should be open. The place for issues like these is in the mailing list in the first instance, as it's almost certainly a configuration/network issue on user's part.

**Comment by andrewdavidwong** [comment]
> @andrewdavidwong Actually this isn't a duplicate because #4778 was about stretch and this is debian-10 (buster) , but I dont think either of them should be open. The place for issues like these is in the mailing list in the first instance, as it's almost certainly a configuration/network issue on user's part.

Thank you for the correction. I'll fix the resolution on this issue and close #4778.
